cache/commands/src/cache.cmd.ts

```diff
@@ -81,6 +82,9 @@ export async function handler (opts: CacheCommandOptions, params: string[]): Pro
       registry: opts.cliOptions['registry'],
     }, params.slice(1))
   case 'view': {
+    if (!params[1]) {
```

Comment:
maybe it should check the length instead. More params than 2 should also probably be an error

Reply:
Should two judgments be added to give two different error prompts?